com.github.bcedu.valasimplehttpserver

bcedu opened the pull request:
Main repo: https://github.com/bcedu/ValaSimpleHTTPServer

## Review Checklist
* [x] App opens
* [x] Does what it says
* [x] Categories match

### AppData
* [x] Name is unique and non-confusing
* [x] https://github.com/bcedu/ValaSimpleHTTPServer/issues/36
* [x] Matches description
* [x] https://github.com/bcedu/ValaSimpleHTTPServer/issues/34
* [x] Matches screenshot
* [x] Launchable tag with matching ID
* [x] Release tag with matching version and YYYY-MM-DD date
* [x] Custom colors meet WCAG A contrast or greater
* [x] OARS info matches

### Flatpak
* [x] Uses elementary runtime
* [x] https://github.com/bcedu/ValaSimpleHTTPServer/issues/35
* [x] Sandbox permissions are reasonable

danrabbit reviewed (changes requested):
Hey @bcedu thanks for your submission! I filed a few issue reports with some required changes here. Mostly nothing too big, just some updates needed

I have a question about the filesystem permissions you've requested here. Can you explain why you have `host` and `device=all`?

```
  - '--filesystem=host'
  - '--device=all'
  - '--filesystem=xdg-documents'
```

I'm also thinking that fixing https://github.com/bcedu/ValaSimpleHTTPServer/issues/37 will probably let you drop the `xdg-documents` hole

Non-blocking comments:

There's a new runtime `6.1`, so I recommend you update from `6` in your flatpak manifest

I'd also recommend checking out the [additional categories](https://specifications.freedesktop.org/menu-spec/latest/apas02.html) list and see if any of these can be added to your Desktop Entry file. I imagine `FileTransfer` and/or `P2P` would apply here

bcedu commented:
Hi!

- I've changed the use of filechooser (https://github.com/bcedu/ValaSimpleHTTPServer/issues/37).

- I removed unnecessary permisions from flatpak yml

- I've updated the runtime to 6.1

danrabbit reviewed (approved):
Thanks for making those adjustments! Works like a charm :)
